packages/menu/src/MenuItem.tsx

```diff
@@ -20,6 +19,10 @@ const containerStyle = css`
   text-decoration: none;
   position: relative;
   transition: background-color 150ms ease-in-out;
+  border: none;
+  margin: unset;
+  width: 100%;
+  font-family: 'Akzidenz';
```

Comment:
We should always set font-family fallbacks. Can we make the font-family list here consistent with our other uses?

```css
font-family: Akzidenz, 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

As an aside, we should think about how to share design tokens like this in the future.